daemon/CryptoImpl/cryptorequestprocessor.cpp

```diff
@@ -88,63 +90,6 @@ Daemon::ApiImpl::RequestProcessor::RequestProcessor(
             this, &Daemon::ApiImpl::RequestProcessor::secretsCryptoPluginLockCodeRequestCompleted);
 }
 
-bool
-Daemon::ApiImpl::RequestProcessor::loadPlugins()
-{
-    // First, see if any of the EncryptedStorage plugins from Secrets are also
-    // Crypto plugins (providing generateAndStoreKey() functionality internally).
-    qCDebug(lcSailfishCryptoDaemon) << "Loading crypto storage plugins";
-    QMap<QString, QObject*> potentialCryptoPlugins = m_secrets->potentialCryptoStoragePlugins();
```

Comment:
Does the new code handle these correctly?  A quick grep didn't find any calls to potentialCryptoStoragePlugins() in the new code, but I might have missed something.

Reply:
These plugins are loaded and should be returned by both `getPlugins<CryptoPlugin>()` and `getPlugins<EncryptedStoragePlugin>()` since they can be `qobject_cast()`ed to both. Is there any more special handling necessary?

Reply:
Yes.  See the threading PR for a good example: we need to know which Crypto plugins are also EncryptedStoragePlugins so that we can run operations interacting with those plugins on the appropriate (in that case, Secrets) thread.  There may be other cases in the code which currently call the potentialCryptoStoragePlugins() method also.